apisix/plugins/limit-count/limit-count-local.lua

```diff
@@ -14,7 +14,34 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
-local limit_local_new = require("resty.limit.count").new
+local limit_count = require("resty.limit.count")
+
+limit_count.handle_incoming = function (self, key, cost, commit)
+  if cost < 1 then
+    return nil, "cost must be at least 1"
+  end
+  local dict = self.dict
+  local limit = self.limit
+  local window = self.window
+
```

Comment:
I think we'd better not hack the underlying functions and suggest directly fork the corresponding repositories to make changes

Reply:
Nevertheless, we HAVE to maintain a separate codebase. And in fact, we are just adding a new function which is essentially a modified version of the existing `incoming` function.